header.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
You shouldn't be re-declaring the doctype in the middle of the document, the same goes for the <html> tags.